4.2-dev/base-how-to.md

```diff
@@ -286,7 +286,7 @@ Your result should say "_webpack compiled successfully_". If so, you can now use
 ```diff
     'styles' => [
 -        'packages/backpack/base/css/bundle.css',
-+        'css/custom-backpack-bundle.css',
++        'packages/backpack/base/css/custom-backpack-bundle.css',
```

Comment:
```suggestion
+        'css/custom-backpack-bundle.css',
```

Reply:
Actually our point here was to NOT publish anything in the `packages/backpack` directory, but to publish it in your own directory.